projects/demo-playwright/utils/page-objects/input-card-grouped.po.ts

```diff
@@ -0,0 +1,42 @@
+import {Locator, Page} from '@playwright/test';
+
+export class TuiInputCardGroupedPO {
+    readonly cardGroupInput = this.page.locator(
+        `[automation-id="tui-input-card-grouped__card"]`,
+    );
+
+    readonly cleanerIcon = this.page.locator(
+        `[automation-id="tui-input-card-grouped__cleaner"]`,
+    );
+
+    readonly disableFormControlToggle = this.page
+        .locator(`tr`)
+        .filter({
+            has: this.page.locator(`[automation-id="tui-documentation__property-name"]`, {
+                hasText: `disabled`,
+            }),
+        })
+        .locator(`[automation-id="tui-toggle__checkbox"]`);
+
+    constructor(private readonly page: Page) {}
```

Comment:
See other PO examples and make `TuiInputCardGroupedPO` to be similar to them

https://github.com/taiga-family/taiga-ui/blob/0e61adf8a57b87415879792cd856b65531c265e0/projects/demo-playwright/utils/page-objects/input-date-range.po.ts#L7